projects.yml

```diff
@@ -4607,4 +4607,10 @@ graphqlize:
   name: GraphQLize
   url: https://www.graphqlize.org
   categories: [GraphQL]
-  platforms: [clj]
\ No newline at end of file
+  platforms: [clj]
+
+pogonos:
+  name: Pogonos
+  url: https://github.com/athos/pogonos
+  categories: [Template Languages]
+  platform: [clj, cljs]
```

Comment:
`platform` should be `platforms`, plural.

Reply:
My bad! Just fixed it.